src/rpc/rpcquorums.cpp

```diff
@@ -288,12 +288,13 @@ UniValue quorum_memberof(const JSONRPCRequest& request)
 void quorum_sign_help()
 {
     throw std::runtime_error(
-            "quorum sign llmqType \"id\" \"msgHash\"\n"
+            "quorum sign llmqType \"id\" \"msgHash\" \"quorumHash\"\n"
```

Comment:
I think optional params generally have parens around them here to show that it's not required.